v2g-liberty-cards/src/util/utils_time.ts

```diff
@@ -0,0 +1,14 @@
+export function elapsedTimeSince(dateTimeStamp: string) {
```

Comment:
Suggestion: rename to `time.ts` (as originally proposed).
The folder is already called `util/` so adding `utils_` another time is redundant.

Reply:
Ah, oversaw that. Will do.